src/sql/changes/2025-06-02-add-totp-tables.sql

```diff
@@ -0,0 +1,23 @@
+-- Add TOTP authentication tables
+
+-- TOTP secrets with verification state
+CREATE TABLE user_totp (
+    user_id INTEGER PRIMARY KEY REFERENCES users(user_uid) ON DELETE CASCADE,
+    secret TEXT NOT NULL,
+    verified BOOLEAN DEFAULT FALSE,
+    created_at TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
+    updated_at TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+);
+
+-- Backup codes (8-character alphanumeric)
+CREATE TABLE user_backup_codes (
+    id SERIAL PRIMARY KEY,
```

Comment:
Do we need to have an `id` column here? It seems like we can make `user_id` the `PRIMARY KEY` here like in the `user_totp` table.

Reply:
We need multiple rows per `user_id` (foreign key) as each user has up to 10 backup codes.

Reply:
Sounds good!